src/main/java/core/basesyntax/MatrixOperation.java

```diff
@@ -6,18 +6,43 @@
  */
 public class MatrixOperation {
     public int[] getDiagonal(int[][] matrix) {
-        return null;
+        int[] mainDiagonal = new int[matrix.length];
+        for (int indexOfMatrix = 0; indexOfMatrix < matrix.length; indexOfMatrix++) {
```

Comment:
```suggestion
        for (int i = 0; indexOfMatrix < matrix.length; indexOfMatrix++) {
```
you can use "i" in loops, that's common practice

Reply:
Done